<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FsDialog</class>
 <widget class="QDialog" name="FsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConcSetting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel,QSpinBox,QDoubleSpinBox,QPushButton,QLineEdit,QComboBox{
	min-height:48px;
	font: 12pt &quot;黑体&quot;;
}

QGroupBox{
	font: Bold 14pt &quot;黑体&quot;;
}

QDialog{
	background-color: rgb(255, 255, 255);
}

/*spinbox 抬起样式*/
QTimeEdit::up-button,QDoubleSpinBox::up-button,QSpinBox::up-button {subcontrol-origin:border;
    subcontrol-position:right;
	image: url(:/icons/up_left.png);
	background-color: rgb(255, 255, 255);
    width: 36px;
    height: 50px;       
}
QTimeEdit::down-button,QDoubleSpinBox::down-button,QSpinBox::down-button {subcontrol-origin:border;
    subcontrol-position:left;
	image: url(:/icons/up_right.png);
	background-color: rgb(255, 255, 255);
    width: 36px;
    height: 50px;
}

/*按钮按下样式*/
QTimeEdit::up-button:pressed,QDoubleSpinBox::up-button:pressed,QSpinBox::up-button:pressed{subcontrol-origin:border;
    subcontrol-position:right;
    image: url(:/icons/up_left.png);
	background-color: rgb(255, 255, 255);
    width: 35px;
    height: 49px;       
}
  
QTimeEdit::down-button:pressed,QDoubleSpinBox::down-button:pressed,QSpinBox::down-button:pressed,QSpinBox::down-button:pressed{
    subcontrol-position:left;
    image: url(:/icons/up_right.png);
	background-color: rgb(255, 255, 255);
    width: 35px;
    height: 49px;   
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Setting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
        <item>
         <widget class="QLabel" name="concLabel">
          <property name="text">
           <string>Conc</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="concDoubleSpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>99999999999999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
        <item>
         <widget class="QLabel" name="unitLabel">
          <property name="text">
           <string>Unit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="unitComboBox">
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="cancelPBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okPBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
